src/stores/ProjectStore.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const alt = require('../alt');
+
+const ProjectActions = require('../actions/ProjectActions.js');
+
+class ProjectStore {
+  constructor() {
+
+    this.bindActions(ProjectActions);
+
+    this.state = {
+      projectName: ''
+    };
+
+  }
+
+  onClearName() {
+    this.setState({
+      projectName: ''
+    });
+  }
+
+  onUpdateName(evt) {
+    this.setState({
+      projectName: evt.target.value
```

Comment:
pull `value` out of the event before calling the action, I don't like events being passed through the dispatcher into the store, as it is a UI concern, not a data concern